database/console/migrate_command_test.go

```diff
@@ -25,148 +21,18 @@ func TestMigrateCommand(t *testing.T) {
 		t.Skip("Skipping tests of using docker")
 	}
 
-	var (
-		mockConfig *configmock.Config
-		query      ormcontract.Query
-	)
+	testQueries := gorm.NewTestQueries().Queries()
```

Comment:
Don't distinguish drivers, get test Queries from a unique method, to improve the testing speed when using the minimum test model.